Sources/SwiftGenKit/Parsers/FontsParser.swift

```diff
@@ -56,7 +56,7 @@ extension CTFont {
     let descs = CTFontManagerCreateFontDescriptorsFromURL(file.url as CFURL) as NSArray?
     guard let descRefs = (descs as? [CTFontDescriptor]) else { return [] }
 
-    return descRefs.flatMap { desc -> Font? in
+    return descRefs.compactMap { desc -> Font? in
```

Comment:
(… As compared to here where the change is justified as we operate on an array but the closure returns an optional, so here it's indeed intended to be the non-monadic `compactMap` and this change is expected)